crossing-training.lic

```diff
@@ -382,15 +382,36 @@ class CrossingTraining
   def calculate_mana(min, more, discern_data, cyclic)
     total = min + more
     total = (total * @settings.prep_scaling_factor).floor
-    discern_data['mana'] = [(total / 4.0).ceil, min].max
+    discern_data['mana'] = [(total / 5.0).ceil, min].max
     remaining = total - discern_data['mana']
-    if remaining > @settings.cambrinth_cap
+    total_cambrinth_mana = 0
+    @settings.cambrinth_items.each do |item|
+      total_cambrinth_mana += item[:cap]
```

Comment:
I prefer the 'string' style (`item['cap']`) over symbol style (`item[:cap]`) myself. I think it makes the YAML file a bit easier to read (and is definitely more common).